packages/diagram_generator/android/app/build.gradle

```diff
@@ -1,71 +1,44 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Here and elsewhere: Do we need to keep these license headers?

Reply:
The license check doesn't fail, and they're all in the generated code. I added them back in already to the ones that failed the license check. I'm assuming no.